jekyll/_cci2/language-android.md

```diff
@@ -82,71 +106,8 @@ jobs:
           path: app/build/test-results
       # See https://circleci.com/docs/2.0/deployment-integrations/ for deploy examples
 ```
-
 {% endraw %}
 
-## Config walkthrough
-
-We always start with the version.
-
-```yaml
-version: 2
-```
-
-Next, we have a `jobs` key. Each job represents a phase in your Build-Test-Deploy process. Our sample app only needs a `build` job, so everything else is going to live under that key.
-
-In each job, we have the option of specifying a `working_directory`. This is the directory into which our code will be checked out, and this path will be used as the default working directory for the rest of the `job` unless otherwise specified.
-
-```yaml
-jobs:
-  build:
-    working_directory: ~/code
-```
-
-Directly beneath `working_directory`, we can specify container images under a `docker` key.
-
-```yaml
-    docker:
-      - image: circleci/android:api-25-alpha
-        auth:
-          username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-```
-
-We use the CircleCI-provided Android image with the `api-25-alpha` tag. See [Docker Images](#docker-images) below for more information about what images are available.
-
-Now we’ll add several `steps` within the `build` job.
-
-We start with `checkout` so we can operate on the codebase.
-
-Next we pull down the cache, if present. If this is your first run, or if you've changed either of your `build.gradle` files, this won't do anything. We run `./gradlew androidDependencies` next to pull down the project's dependencies. Normally you never call this task directly since it's done automatically when it's needed, but calling it directly allows us to insert a `save_cache` step that will store the dependencies in order to speed things up for next time.
-
-Then `./gradlew lint test` runs the unit tests, and runs the built in linting tools to check your code for style issues.
-
-We then upload the build reports as job artifacts, and we upload the test metadata (XML) for CircleCI to process.
-
-## Docker images
-
-For convenience, CircleCI provides a set of Docker images for building Android apps. These pre-built images are available in the [CircleCI org on Docker Hub](https://hub.docker.com/r/circleci/android/). The source code and Dockerfiles for these images are available in [this GitHub repository](https://github.com/circleci/circleci-images/tree/master/android).
```

Comment:
I'm not sure if we want to retain some of this info and move it to be under `Sample configuration for unit tests`. I'm fine either way!

Reply:
True, I will add these back in. Thanks for catching this.